### ISSUE TITLE
Mac OS Sierra support

### ISSUE THREAD

**Issue by NicolasRannou** [powerline/fonts#185]
Does it work on MacOS Sierra?

It is the first time I try powerline and I do not know if I didn't install it properly or if it is a macOS Sierra issue:

![screen shot 2016-08-09 at 1 06 39 pm](https://cloud.githubusercontent.com/assets/214063/17514300/21b34dc4-5e32-11e6-8329-93fab94b300a.png)

<img width="823" alt="screen shot 2016-08-09 at 1 06 52 pm" src="https://cloud.githubusercontent.com/assets/214063/17514306/2955f176-5e32-11e6-8c64-3e7ae595b0bc.png">

**Comment by NicolasRannou** [comment]
We need to set the terminal font to a powerline font.
![screen shot 2016-08-09 at 1 16 53 pm](https://cloud.githubusercontent.com/assets/214063/17514563/928ea150-5e33-11e6-933c-dc752aa95880.png)

**Comment by landsman** [comment]
Oh, thanks for it!
Can you add this to read?
I think That we are not only idiots sole Who will think about this.

**Comment by artipfw** [comment]
any solution?

**Comment by jjalonso** [comment]
not working here

**Comment by SarkarSubhajit** [comment]
Got same issue here. Just change the font to Droid Sans Mono Awesome and it's working fine.

**Comment by daftskunk** [comment]
SarkarSubjhajit's answer worked fine for me! Everyother font from Powerline didn't work.

**Comment by jbanulso** [comment]
If this problem appears you should check the option "Use a different font for non-ASCII text", and select a font that works - i.e. any of the Meslo (not Menlo) family.

**Comment by Ghosty141** [comment]
<img width="1187" alt="screen shot 2017-07-17 at 19 08 42" src="https://user-images.githubusercontent.com/17564201/28281539-be719276-6b27-11e7-9025-fb8c59c349c4.png">

@jbanulso Somehow doesn't work for me. It seems like my terminal is cursed...

**Comment by jbanulso** [comment]
@Ghosty141 hmm interesting... Have you checked the last part in the README? It says the following: 

> In some distributions, Terminess Powerline is ignored by default and must be explicitly allowed. A fontconfig file is provided which enables it. Copy this file from the fontconfig directory to your home folder under `~/.config/fontconfig/conf.d` (create it if it doesn't exist) and re-run `fc-cache -vf`.

Maybe that can end the curse :)

**Comment by Ghosty141** [comment]
@jbanulso I forgot the most important thing when things don't work: Did you try turning it off and on again? It was fixed after a restart, kinda weird since on my hackintosh it worked right away. Anyway, thanks for the help!

**Comment by sergioviniciuss** [comment]
After upgrading to high sierra the icons stopped working for me, so the solution I found was to follow the steps provided in here: https://github.com/gabrielelana/awesome-terminal-fonts/wiki/OS-X

I hope it helps if someone else is still having this issue.

**Comment by BobEve** [comment]
@NicolasRannou word for me.thank you!

**Comment by bjoseg** [comment]
@SarkarSubhajit works for me! thanks!!

**Comment by polpenaloza** [comment]
If you have a pre-determined window arrangement, you will need to re-arrange it again.

TL;DR: create a new arrangement.

**Comment by zuxfoucault** [comment]
Do the powerline symbols include in each "font for powerline"? Or I have to download PowerlineSymbols independently?

**Comment by testacode** [comment]
Couldn't install them, I won't disable any security on my system to install a font. Isn't there any other solution appart from not using the theme on zsh?

**Comment by yairEO** [comment]
MacOS won't let me click the button for changing font. clicking does nothing :/

**Comment by mandarvaze** [comment]
FWIW, Latest version of iTerm2 (Build 3.3.0beta12) has a setting "Use built-in Powerline Glyphs" (I'm on macOS mojave) - Worked for me.

**Comment by raubin** [comment]
@mandarvaze - On 3.3.2 this checkbox is there as well and it worked perfectly. Thank you!

**Comment by vjosullivan** [comment]
The iTerm2 now supports Powerline glyphs natively.  No need to install any fonts.
**iTerm2 > Preferences > Profiles > Text > Use built-in Powerline glyphs**

**Comment by mikeseal** [comment]
> @jbanulso I forgot the most important thing when things don't work: Did you try turning it off and on again? It was fixed after a restart, kinda weird since on my hackintosh it worked right away. Anyway, thanks for the help!

Yep, spent a hour messing around and after trawling a few site I find a reboot is required.  I went into a cold sweat as I thought we were back in MS Windows land.
